src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -218,6 +196,93 @@
         }
     }
 
+    private ResponseEntity<?> getListedSubscriptions(String subscriptionName) {
+        // set is used to prevent subscription names repeating
+        Set<String> subscriptionNameList = new HashSet<>(Arrays.asList(subscriptionName.split(",")));
+        List<Subscription> foundSubscriptionList = new ArrayList<>();
+        List<String> notFoundSubscriptionList = new ArrayList<>();
+
+        subscriptionNameList.forEach(name -> {
+            try {
+                LOGGER.debug("Subscription fetching has been started: {}", name);
+                // Make sure the password is not sent outside this service.
+                Subscription subscription = subscriptionService.getSubscription(name);
+                subscription.setPassword("");
+                foundSubscriptionList.add(subscription);
+                LOGGER.debug("Subscription [{}] fetched successfully.", name);
+            } catch (SubscriptionNotFoundException e) {
+                LOGGER.error("Subscription not found: {}", name);
```

Comment:
Subscription not found is not a error, I guess. It is just that the subscription don't exist in MongoDb. Should it be more a debug or info message

Reply:
Hmm I guess that's true. There isn't anything technically wrong with not finding a subscription so I guess this could be a debug message.

Reply:
SubscriptionNotFoundException now logs as debug

Reply:
ok